db.py

```diff
@@ -230,20 +230,28 @@ def serialize(self):
 		}
 
 	def comment_serialize(self):
-		self.db.cur.execute(
-			"""
-SELECT comments.user_id, post_id, content
-	FROM comments
-	JOIN posts ON post_id = posts.id
-	WHERE posts.id = %s;""", (self.id,)
-		)
+    self.db.cur.execute(
+        """
+  SELECT comments.user_id, post_id, content
+    FROM comments
+    JOIN posts ON post_id = posts.id
+    WHERE posts.id = %s;""", (self.id,)
+      )
+
+      rows = self.db.cur.fetchall()
+
+      return {
+        "comments": [row[2] for row in rows],
+        "users": [row[0] for row in rows]
+      }
+		
 
-		rows = self.db.cur.fetchall()
 
-		return {
-			"comments": [row[2] for row in rows],
-			"users": [row[0] for row in rows]
-		}
+
+class DatabaseComments:
```

Comment:
What purpose does this class serve?

Reply:
It might be better represented by having a `comments` function in `DatabasePost` which returns `DatabaseComment` objects. It's pretty unconventional to store a list of users and content strings if they're supposed to be zipped together.

Reply:
This class is being used to return the comments function that is under database class. If we remove this class the comment feature breaks.